### DISCUSSION TITLE
Installation issue caused by code analyzers

### DISCUSSION THREAD

**Discussion by MacMiggity** [nxrighthere/UnrealCLR#186] [Help] [answered]
I'm getting this error regarding these packages when running dotnet run inside the install directory. 

```
dotnet --version
5.0.401
```

I tried installing every bit of toolchain for C++ and C# that I can find in my Visual Studio Installer to help with this, but reaching an impasse. Instead of burning a bunch of time I thought I would reach out and see if anyone has experienced similar issues, and maybe I'm missing something simple, but if there are steps to remedy. 

Thanks,

```
→ D:\Code\Random\UnrealCLR-1.30\Install› dotnet run
D:\Code\Random\UnrealCLR-1.30\Install\Install.csproj : error NU1101: Unable to find package Microsoft.CodeAnalysis.FxCopAnalyzers. No packages exist with this id in source(s): Microsoft Visual Studio Offline Packages
D:\Code\Random\UnrealCLR-1.30\Install\Install.csproj : error NU1101: Unable to find package Microsoft.CodeQuality.Analyzers. No packages exist with this id in source(s): Microsoft Visual Studio Offline Packages
D:\Code\Random\UnrealCLR-1.30\Install\Install.csproj : error NU1101: Unable to find package Microsoft.NetCore.Analyzers. No packages exist with this id in source(s): Microsoft Visual Studio Offline Packages

The build failed. Fix the build errors and run again.
```

**Comment by nxrighthere** [comment, marked as answer]
Try to use the latest version from the master branch.

**Reply by nxrighthere** [reply]
Also, it seems that only an offline package source used to install NuGet packages, you have to fix that in Visual Studio.

**Reply by MacMiggity** [reply]
thanks
